<launch>

    <!--这里尽量不要出现中文路径-->
    <!--bin->npy时,由于bin的读取速度很快,导致在pub的时候python节点还没有初始化完成,导致没有转换-->

    <arg name="input_dir" default="/media/cyber-z/E/test_trans/pcd/1" /> 
    <arg name="out_dir" default="/media/cyber-z/E/test_trans/npy/1" /> 
    <arg name="input_type_option" default="pcd" />   <!--  pcd  -->
    <arg name="start_num" default="1" />
    <arg name="trans_count" default="20" />
    <arg name="pub_topic" default="/test/pointcloud" />
    <arg name="cloud_type" default="XYZ" />      <!--  XYZ/XYZI  预留接口-->

    <node pkg="pointcloud_transfer" type="tranp_c2p_node.py" name="tranp_c2p_node" output="screen">

        <param name="out_dir_" type="string" value="$(arg out_dir)" />
        <param name="start_num_" type="int" value="$(arg start_num)" />
        <param name="pub_topic_p" type="string" value="$(arg pub_topic)" />
        <param name="cloud_type_" type="string" value="$(arg cloud_type)" />

    </node>

    <node pkg="pointcloud_transfer" type="pubc_node" name="pubc_node" output="screen">
        <param name="input_dir_" type="string" value="$(arg input_dir)" />
        <param name="input_type_option_" type="string" value="$(arg input_type_option)"/>
        <param name="trans_count_" type="int" value="$(arg trans_count)" />
        <param name="start_num_" type="int" value="$(arg start_num)" />
        <param name="pub_topic_" type="string" value="$(arg pub_topic)" />
        <param name="cloud_type_" type="string" value="$(arg cloud_type)" />
    </node>





</launch>